website/content/en/docs/building-operators/golang/references/env-test-setup.md

```diff
@@ -0,0 +1,64 @@
+---
+title: EnvTest Setup
+linkTitle: EnvTest Setup
+weight: 50
+---
+
+## Overview 
+
+This document describes how to configure the environment for the [controller tests][controller-test] which uses [envtest][envtest] and is supported by the SDK. 
+
+## Installing prerequisites
+
+[Envtest][envtest] requires that the `kubectl`, `api-server` and `etcd` be present locally. You can use this [script][script] to download these binaries into the `testbin/` directory. It may be convenient to add this script your Makefile as follows:
```

Comment:
```suggestion
[Envtest][envtest] requires that `kubectl`, `api-server` and `etcd` be present locally. You can use this [script][script] to download these binaries into the `testbin/` directory. It may be convenient to add this script your Makefile as follows:
```